metron-platform/metron-common/src/main/java/org/apache/metron/common/stellar/shell/StellarShell.java

```diff
@@ -244,6 +304,55 @@ private void write(String out) {
   }
 
   private void writeLine(String out) {
-    System.out.println(out);
+    console.getShell().out().println(out);
+  }
+
+  @Override
+  public int execute(ConsoleOperation output) throws InterruptedException {
+    String expression = output.getBuffer().trim();
+    if(StringUtils.isNotBlank(expression)) {
+      if(isMagic(expression)) {
+        handleMagic( expression);
+
+      } else if(isDoc(expression)) {
+        handleDoc(expression);
+
+      } else if (expression.equals("quit")) {
+        try {
```

Comment:
Does this prevent the user from referring to a variable called `quit`?  If so, maybe `%quit`?